File: c++/control_flow/if_statement/if_else_statement_formatting.cpp
#include <iostream>

int main()
{
  bool condition;

  condition = (1 > 2);

  if (condition)
    std::cout << "1 > 2" << std::endl;
  else
    std::cout << "!(1 > 2)" << std::endl;

  condition = (2 > 1);

  if (condition)
    std::cout << "2 > 1" << std::endl;
  else
    std::cout << "!(2 > 1)" << std::endl;

  std::cout << std::endl;

  if (1 > 2)
    std::cout << "1 > 2" << std::endl;
  else
    std::cout << "!(1 > 2)" << std::endl;

  if (2 > 1)
    std::cout << "2 > 1" << std::endl;
  else
    std::cout << "!(2 > 1)" << std::endl;

  std::cout << std::endl;

  if (1 > 2) std::cout << "1 > 2" << std::endl;
  else std::cout << "!(1 > 2)" << std::endl;

  if (2 > 1) std::cout << "2 > 1" << std::endl;
  else std::cout << "!(2 > 1)" << std::endl;

  std::cout << std::endl;

  if (1 > 2)
  {
    std::cout << "1 > 2" << std::endl;
  }
  else
  {
    std::cout << "!(1 > 2)" << std::endl;
  }

  if (2 > 1)
  {
    std::cout << "2 > 1" << std::endl;
  }
  else
  {
    std::cout << "!(2 > 1)" << std::endl;
  }

  std::cout << std::endl;
}

File: c#/control_flow/if_statement/IfElseStatementFormatting.cs
class IfElseStatementFormatting {
  private static void Main(string[] args) {
    bool condition;

    condition = (1 > 2);

    if (condition)
      System.Console.WriteLine("1 > 2");
    else
      System.Console.WriteLine("!(1 > 2)");

    condition = (2 > 1);

    if (condition)
      System.Console.WriteLine("2 > 1");
    else
      System.Console.WriteLine("!(2 > 1)");

    System.Console.WriteLine();

    if (1 > 2)
      System.Console.WriteLine("1 > 2");
    else
      System.Console.WriteLine("!(1 > 2)");

    if (2 > 1)
      System.Console.WriteLine("2 > 1");
    else
      System.Console.WriteLine("!(2 > 1)");

    System.Console.WriteLine();

    if (1 > 2) System.Console.WriteLine("1 > 2");
    else System.Console.WriteLine("!(1 > 2)");

    if (2 > 1) System.Console.WriteLine("2 > 1");
    else System.Console.WriteLine("!(2 > 1)");

    System.Console.WriteLine();

    if (1 > 2)
    {
      System.Console.WriteLine("1 > 2");
    }
    else
    {
      System.Console.WriteLine("!(1 > 2)");
    }

    if (2 > 1)
    {
      System.Console.WriteLine("2 > 1");
    }
    else
    {
      System.Console.WriteLine("!(2 > 1)");
    }

    System.Console.WriteLine();
  }
}
